cloud_experts_tutorials/cloud-experts-aws-secret-manager.adoc

```diff
@@ -17,38 +17,49 @@ toc::[]
 //   - Chris Kang
 // ---
 
-The AWS Secrets and Configuration Provider (ASCP) provides a way to expose AWS Secrets as Kubernetes storage volumes. With the ASCP, you can store and manage your secrets in Secrets Manager and then retrieve them through your workloads running on ROSA.
-
-This is made even easier and more secure through the use of AWS STS and Kubernetes PodIdentity.
+The AWS Secrets and Configuration Provider (ASCP) provides a way to expose AWS Secrets as Kubernetes storage volumes. With the ASCP, you can store and manage your secrets in Secrets Manager and then retrieve them through your workloads running on {product-title} (ROSA).
 
 [id="cloud-experts-aws-secret-manager-prerequisites"]
 == Prerequisites
 
+Ensure that you have the following resources and tools before starting this process:
+
 * A ROSA cluster deployed with STS
 * Helm 3
-* aws CLI
-* oc CLI
-* jq
+* `aws` CLI
+* `oc` CLI
+* `jq` CLI
 
+[discrete]
 [id="cloud-experts-aws-secret-manager-preparing-environment"]
-== Preparing Environment
+=== Additional environment requirements
 
-. Validate that your cluster has STS:
+. Log in to your ROSA cluster by running the following comnmand:
++[source,terminal]
+----
+$ oc login --token=<your-token> --server=<your-server-id>
+----
++
+You can find your log in token by accessing your cluster in {cluster-manager-url-pull}.
```

Comment:
From the preview link, I see below effect:
![Screenshot from 2024-01-08 20-47-53](https://github.com/openshift/openshift-docs/assets/13775942/1ed67b80-afc0-44c0-bbbc-97665be060f3)

```
1. Log in ... comnmand: +[source,terminal]
+ You can find your log ...
1. Validate that your cluster has STS ...
```
That is, it has problems:

1. `+[source,terminal]` is shown bad format.
2. Typo `comnmand`
3. `+ You can find your log` has bad format `+`
4. Two items of same starting "1." order.

@EricPonvelle  Suggest document peers review and find such format issues and fix them first before requesting QE review :)